src/utils/__tests__/recipient-test.js

```diff
@@ -49,6 +50,28 @@ describe('normalizeRecipientsAsUserIdsSansMe', () => {
   });
 });
 
+describe('pmKeyRecipientsFromIds', () => {
+  const allUsersById = new Map([eg.selfUser, eg.otherUser, eg.thirdUser].map(u => [u.user_id, u]));
+  const [self, other, third] = [eg.selfUser, eg.otherUser, eg.thirdUser];
+  // prettier-ignore
+  /* eslint-disable no-multi-spaces */
+  /* eslint-disable array-bracket-spacing */
+  for (const [desc, users, expectedSet] of [
```

Comment:
nit: Maybe `description` instead of `desc`?; For a split-second I thought of `descending` because that's another thing that comes to mind when I look at arrays (though I may be biased from all the recent stuff about sorting).